docs/DeveloperGuide.md

```diff
@@ -176,6 +178,95 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add student feature: `add -s`
+
+#### Implementation
+
+The 'add student' feature adds a student contact to TutorAid. A student contact consists of the student's name, 
+student's contact number, the parent's name and parent's contact number. 
+
+The feature is mainly implemented by the following methods:
+* `AddStudentCommand#execute()` —Adds a `Student` object to TutorAid.
```

Comment:
Just a minor nitpick! I think there should be a space after `-` , i.e. `- Adds` ?